src/calendar/calendar.service.ts

```diff
@@ -78,6 +87,19 @@ export class CalendarService {
       throw new HttpException(`no event for ${id} found`, HttpStatus.NOT_FOUND);
     }
 
+    if (
+      !this.validationService.validateAuthor(
+        oldDocument.author,
+        authorObject.uid,
+        authorObject.platform,
+      )
+    ) {
+      throw new HttpException(
+        `update failed: author doesn't match`,
```

Comment:
would this be better as standard `'`?

Reply:
oh, yes 😅 Dont know why I used these ones..

Reply:
lol no worries 😄